src/js/tabs/pid_tuning.js

```diff
@@ -399,6 +407,30 @@ TABS.pid_tuning.initialize = function (callback) {
             $('.idleMinRpm').hide();
         }
 
+        if (semver.gte(CONFIG.apiVersion, "1.43.0")) {
+            const ratesTypeListElement = $('select[id="ratesType"]'); // generates list
+            let ratesList;
+            ratesList = [
```

Comment:
Unify the declaration and the assignment of `ratesList`, and make it `const` and not `let`

Reply:
Didn't make const because in case of more rates it need to be modified, so i thought it can be left as let. I need to change anyway to const?

Reply:
You can add and remove elements from an array that is `const`. What you can't do is to assign the variable to a new array. So you can define it as `const` and use the `.push();` method over the array in a future to add elements if needed.

Reply:
Ok, perfect, will make it const then 👍